src/help/index.js

```diff
@@ -26,6 +26,8 @@ chrome.runtime.getPlatformInfo(function (info) {
 
 const query = parseQuery(window.location.search);
 if (query.message) {
-  const errorTextEl = document.querySelector('.js-error-message');
+  const errorTextEl = /** @type {HTMLElement} */ (document.querySelector(
```

Comment:
I tried without the casting and it doesn't complain. `textContent` is present in `Element` type, I believe.